pyrefinebio/transcriptome_index.py

```diff
@@ -64,19 +64,19 @@ def search(cls, **kwargs):
 
         parameters:
 
-            salmon_version (str): Eg. salmon 0.13.1
+            salmon_version (str): filter based on the index's salmon version eg. salmon 0.13.1
 
-            index_type (str): Eg. TRANSCRIPTOME_LONG
+            index_type (str): filter based on the type of index eg. TRANSCRIPTOME_LONG
```

Comment:
"`TRANSCRIPTOME_LONG` or `TRANSCRIPTOME_SHORT`. If the average read length of the RNA-Seq sample you want to process is greater than 75 base pairs, use `TRANSCRIPTOME_LONG` otherwise use TRANSCRIPTOME_SHORT`."